packages/react-ui-kit/src/Layout/Theme.tsx

```diff
@@ -18,29 +18,32 @@
  */
 
 /** @jsx jsx */
-import {jsx} from '@emotion/core';
+import {jsx, Theme, ThemeProvider as EmotionThemeProvider} from '@emotion/react';
+
 import React from 'react';
-import {ThemeProvider as EmotionThemeProvider} from 'emotion-theming';
 import {COLOR} from '../Identity/colors';
 import {filterProps} from '../util';
 
+// Apply custom theme to Emotion so TypeScript can pick it up in the css prop
+declare module '@emotion/react' {
```

Comment:
Theme needs to be augmented in order to have correct typing for the css prop theme parameter.

Reply:
https://github.com/emotion-js/emotion/issues/1197